cmd/packaging/packaging.go

```diff
@@ -160,3 +161,16 @@ func runDockerPackaging(path string, packagingFormat string, command []string) {
 		os.Exit(1)
 	}
 }
+
+func getTemplateData(projectName string) map[string]string {
+	return map[string]string{
+		"projectName":         projectName,
+		"strippedProjectName": removeDashesAndUnderscores(projectName),
+		"author":              getAuthor(),
+		"version":             pubspec.GetPubSpec().Version,
+		"description":         pubspec.GetPubSpec().Description,
+		"debDependencies":     strings.Join(linuxPackagingDependencies, ","),
+		"snapDependencies":    strings.Join(linuxPackagingDependencies, "\n      - "),
```

Comment:
This feels like it should be part of the packaging-method specific template itself, not the data model that we feed into it.
Maybe we can add the popular templating functions library 'sprig' to our templating. It has a neat `join` function.
http://masterminds.github.io/sprig/string_slice.html#join
This map would have to change into a `map[string]interface{}` though, to support a `[]string` for `linuxPackagingDependencies`. Or maybe we can use a struct instead of a map.

Reply:
This should be done in some way, but in another PR